src/System.Private.CoreLib/shared/System/TimeZoneInfo.Unix.cs

```diff
@@ -74,10 +74,11 @@ private TimeZoneInfo(byte[] data, string id, bool dstDisabled)
                     }
                 }
             }
-            _displayName = _standardDisplayName;
-
-            GetDisplayName(Interop.Globalization.TimeZoneDisplayNameType.Generic, ref _displayName);
             GetDisplayName(Interop.Globalization.TimeZoneDisplayNameType.Standard, ref _standardDisplayName);
+            if (_baseUtcOffset >= TimeSpan.Zero)
+                _displayName = $"(UTC+{_baseUtcOffset:hh\\:mm}) {_standardDisplayName}";
+            else
+                _displayName = $"(UTC-{_baseUtcOffset:hh\\:mm}) {_standardDisplayName}";
```

Comment:
>UTC- [](start = 34, length = 4)

Is this will be right when have zone with utc offset 0?

Reply:
On windows:
```
...
(UTC-01:00) Cabo Verde Is.
(UTC) Coordinated Universal Time
(UTC+00:00) Casablanca
(UTC+00:00) Dublin, Edinburgh, Lisbon, London
(UTC+00:00) Monrovia, Reykjavik
(UTC+01:00) Amsterdam, Berlin, Bern, Rome, Stockholm, Vienna
...
```
I'll attach the full list on both windows and new-macOS a bit later

Reply:
updated the main post